lib/stages/pulsarPostProcess.cpp

```diff
@@ -188,17 +197,22 @@ void pulsarPostProcess::main_thread() {
         if (!new_frame_fpga_seq_num.has_value())
             return;
 
+        // Initialize data for header info, namely position and frequency labels.
+        // See fill_headers() for bit-packing of freq labels into PSRHeader.eud3.
         for (uint32_t i = 0; i < _num_gpus; ++i) {
             beam_coord[i] = get_beam_coord(in_buf[i], in_buffer_ID[i]);
-            if (_timesamples_per_pulsar_packet == 3125) {
-                thread_ids[i] = tel.to_freq_id(in_buf[i], in_buffer_ID[i]);
-            } else if (_timesamples_per_pulsar_packet == 625) {
-                // In the case of 4 frequencies per packet we convert the stream_id into a
-                // kind of node_id that runs from 0 to 255 for the thread_id.
-                ice_stream_id_t stream_id = ice_get_stream_id_t(in_buf[i], in_buffer_ID[i]);
-                thread_ids[i] =
-                    stream_id.crate_id * 16 + stream_id.slot_id + stream_id.link_id * 32;
-            }
+            thread_ids[i] = tel.to_freq_id(in_buf[i], in_buffer_ID[i]);
+            
+            // the following was commented out by E. Fonseca on 7 September 2020.
+            //if (_timesamples_per_pulsar_packet == 3125) {
+            //    thread_ids[i] = tel.to_freq_id(in_buf[i], in_buffer_ID[i]);
+            //} else if (_timesamples_per_pulsar_packet == 625) {
+            //    // In the case of 4 frequencies per packet we convert the stream_id into a
+            //    // kind of node_id that runs from 0 to 255 for the thread_id.
+            //    ice_stream_id_t stream_id = ice_get_stream_id_t(in_buf[i], in_buffer_ID[i]);
+            //    thread_ids[i] =
+            //        stream_id.crate_id * 16 + stream_id.slot_id + stream_id.link_id * 32;
+            //}
```

Comment:
Just remove code that's no longer used.  `git` is better for tracking code changes.

Reply:
of course, just an old habit of mine; removed.